src/core/public/chrome/chrome_service.tsx

```diff
@@ -58,7 +59,11 @@ export interface ChromeBrand {
 export type ChromeBreadcrumb = EuiBreadcrumb;
 
 /** @public */
-export type ChromeHelpExtension = (element: HTMLDivElement) => () => void;
+export interface ChromeHelpExtension {
+  appName: string;
+  links?: HeaderHelpMenuUIExtraLink[];
+  content?: (element: HTMLDivElement) => () => void;
```

Comment:
From usages, `links` seems like a replacement for `content`, so we may want to define them as mutually exclusive instead of being both optionals? Or is there use cases where a developer may want to use them both?

Reply:
There are use cases for both. For instance, Canvas displays both a documentation link and a custom link that opens a flyout with keyboard shortcuts.
https://github.com/elastic/kibana/pull/50736/files#diff-4b49c85975d24172c5abc437d0114b5d